src/main/java/core/basesyntax/model/dto/impl/WriteToFileImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.model.dto.impl;
+
+import core.basesyntax.model.dto.WriteToFile;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class WriteToFileImpl implements WriteToFile {
+    private static final String FILE_NAME = "Report.csv";
+
+    @Override
+    public void writeToFile(String report) {
+        try (BufferedWriter bufferedWriter
+                     = new BufferedWriter(new FileWriter("Report.csv"))) {
+            bufferedWriter.write(report);
+        } catch (IOException e) {
+            throw new RuntimeException("File not found" + FILE_NAME);
```

Comment:
```suggestion
            throw new RuntimeException("File not found" + FILE_NAME, e);
```
Please do not ignore the root cause exception. If you ignore it - you will not be able to read a whole stacktrace